Gemfile

```diff
@@ -50,6 +50,8 @@ gem 'bootsnap', require: false
 
 gem 'rspec-rails'
```

Comment:
- [**Optional**] Even though all the tests you have written for this milestone are passing, some of your model test units are failing when I generally run just `rspec` on the root directory without getting into the requests folder. Kindly make sure that all your test cases are passing.

![Screenshot from 2022-09-22 21-16-27](https://user-images.githubusercontent.com/63087491/191843565-b6228925-cedf-4945-86c6-526b362507b4.png)